src/main/java/com/conveyal/gtfs/GTFSFeed.java

```diff
@@ -707,6 +709,26 @@ public double getTripSpeed (String trip_id, boolean straightLine) {
         return distance / time; // meters per second
     }
 
+    /** Returns sorted map of stop_ids mapped to tuple keys of trip_id, stop_sequence (for use in feed.stop_times) */
+    public SortedSet<Fun.Tuple2<String, Fun.Tuple2>> getStopTimesForStop (String stop_id) {
```

Comment:
Maybe.  I had also considered that, but it seemed that it might be helpful to have the tuple with trip ID.  But now that I'm thinking about it again... you're right (trip ID and sequence are easy enough to pull out from StopTime objects themselves).